README.md

```diff
@@ -55,19 +56,21 @@ It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controlle
 which provides a reconcile function responsible for synchronizing resources untile the desired state is reached on the cluster 
 
 ### Test It Out
-1. Install the CRDs into the cluster:
-
+* Install the CRDs and the operator-controller into a new [KIND cluster](https://kind.sigs.k8s.io/):
 ```sh
-make install
+make run
 ```
+This will build a local container image of the operator-controller, create a KIND cluster and then install onto that KIND cluster.
+This will also deploy the catalogd, rukpak and cert-manager dependencies.
 
-2. Run your controller (this will run in the foreground, so switch to a new terminal if you want to leave it running):
+**OR**
 
+* Create deplpoyment files
 ```sh
-make run
+make quickstart VERSION=0.1.0
 ```
-
-**NOTE:** You can also run this in one step by running: `make install run`
+This will generate a `operator-controller.yaml` file used to deploy the operator-controller, and a `install.sh` script to
+deploy it along with the necessary dependencies (catalogd, rukpak, and cert-manager).
```

Comment:
I'm kind of curious, is there a reason we feel a new contributor would even need this information? My understanding is that this section should just be focused on how to test out your changes on cluster. IMO just the  `make run` should suffice. I can't think of any scenarios where a new contributor would want to use anything other than that unless they already have an understanding of what the project does.

Reply:
I was thinking the same, but left it in place of `make install`, which has inconsistent behavior vs `make uninstall`.